codemodel/model/common/schema.ts

```diff
@@ -30,6 +38,12 @@ export interface Schema extends Aspect {
   /** per-serialization information for this Schema  */
   serialization?: SerializationFormats;
 
+  /** contexts in which the schema is used */
+  contexts?: SchemaContext[];
+
+  /** Known media types in which this schema can be serialized */
+  knownMediaTypes?: string[];
```

Comment:
there is a `KnownMediaType`  enum in `@azure-tools/codegen`

Reply:
Ahhh yes, good call.